### ISSUE TITLE
The console command "export" is missing

### ISSUE THREAD

**Issue by jlemale** [lexik/LexikTranslationBundle#306]
Hi,

First of all thanks for the PR Symfony4 support.
I've just noticed that the console command "export" is not available with `php bin/console list`, only the import one is displayed.

```
 lexik
  lexik:translations:import               Import all translations from flat files (xliff, yml, php) into the database.
```

I've checked the code and it's just missing from the file

> LexikTranslationBundle/Resources/config/services.xml

I've never done a PR myself, that's why I'm not fixing it instead of creating the issue. Sorry about that.

**Comment by eugenekurasov** [comment]
Thanks,

I will be create PR at end of week with fix for ExportCommand.

**Comment by kick-the-bucket** [comment]
This issue was fixed in #314
